### PR TITLE
fix: incorrect object key names for CFN

### PR DESCRIPTION
**Pull request by jhockett**
Also fix hardcoded service name reference

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/lib/push-resources.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ function packageResources(context, resources) {
 
   const packageResource = (context, resource) => {
     let s3Key;
-    return (resource.service === 'LambdaFunction' ? buildResource(context, resource) : packageLayer(context, resource))
+    return (resource.service === FunctionServiceName.LambdaFunction ? buildResource(context, resource) : packageLayer(context, resource))
```
Comment (edwardfoyle) [inline packages/amplify-provider-awscloudformation/lib/push-resources.js:229]: I think this should stay as the enum?

File: packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js [hunk 2]
```diff
@@ -53,7 +52,7 @@ const addTrigger = async triggerOptions => {
     throw new Error('Function plugin not installed in the CLI. You need to install it to use this feature.');
   }
 
-  await add(context, 'awscloudformation', FunctionServiceName.LambdaFunction, {
+  await add(context, 'awscloudformation', 'LambdaFunction', {
```
Comment (edwardfoyle) [inline packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js:55]: I think this needs to be the enum

File: packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js [hunk 3]
```diff
@@ -136,7 +135,7 @@ const updateTrigger = async triggerOptions => {
     await update(
       context,
       'awscloudformation',
-      FunctionServiceName.LambdaFunction,
+      'LambdaFunction',
```
Comment (edwardfoyle) [inline packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js:138]: same here

File: packages/amplify-provider-awscloudformation/lib/push-resources.js (hunk 1 above)
Comment (jhockett) [reply packages/amplify-provider-awscloudformation/lib/push-resources.js:229]: I'm changing this to the enum here

File: packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js (hunk 2 above)
Comment (jhockett) [reply packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js:55]: You're correct, thanks for catching this

**Pull request merged**